plugins/events-manager/templates/events-list.php

```diff
@@ -13,8 +13,11 @@
 ?>
 
 <?php
+	$theme_directory = get_template_directory_uri();
```

Comment:
It looks like this is an unused variable?

Reply:
Good catch! Thanks @jveens